docs/intro.rst

```diff
@@ -84,18 +92,32 @@ Windows, it's a druid!)
 
 Caldera got a lot of patents for the installer, including being able
 to play a game during installation which is, incidentally, why we
-never did that in anaconda.
+never did that in Anaconda.  Well, there's also the fact that
+folks at Red Hat took trademarks very `seriously <https://bugzilla.redhat.com/show_bug.cgi?id=224627>`_,
+and worked hard to avoid shipping a game of falling blocks under
+that namme.
 
 OK, so Caldera has made Lizard.  Now Red Hat needed to act.  They
 created a new installer project to focus on this graphical installer.
-The installer was already in Python at this point, but was not
-graphical.  Red Hat started working on this.  They went with the name
-"anaconda" because (a) it is written in Python and an anaconda snake
-is a type of python and (b) the anaconda snake eats lizards in the
-wild.
+The installer up to this point was all written in C, and development
+was relatively slow due to the low level of the language.  To move
+fast, Erik Troan decided to build a new installer with the UI workflows
+written in Python.  Erik went with the name "Anaconda" because (a) it
+is written in Python and an Anaconda snake is a type of python and
+(b) the Anaconda snake eats lizards in the wild.  It was one of a long
+line of creatively named software modules that Erik wrote as part of the
+installer.  There was
+"`pump <https://archive.download.redhat.com/pub/redhat/linux/6.2/en/os/i386/misc/src/anaconda/pump/>`_",
+a bootp / DHCP client library, so named because (a) A pump is a kind
+of shoe, like a `boot[p] <https://manpages.ubuntu.com/manpages/bionic/man8/pump.8.html#quibble>`_
+and (b) "plumb"ing up a network interface is like priming a "pump".
+And there was
+"`balkan <https://archive.download.redhat.com/pub/redhat/linux/6.2/en/os/i386/misc/src/anaconda/balkan/>`_",
+a library to handle partition tables, so named because the Balkans have
+always had to deal with partitioning.
 
-And that's how we got the name anaconda for the installer.  Caldera
+And that's how we got the name Anaconda for the installer.  Caldera
 eventually disappeared entirely.  OpenLinux didn't go anywhere.  But
-Caldera really tried.  The last thing they did?  They bought SCO and
+Caldera really tried.  The last think they did?  They bought SCO and
```

Comment:
I am not sure about `think` in this sentence, `thing` makes more sense to me.

Reply:
Oops! Fixed in an amended commit that I just pushed to the same branch name